uws/UWS/models.py

```diff
@@ -1,6 +1,34 @@
 # -*- coding: utf-8 -*-
 from lxml import etree as et
 
+uws_1_namespace = "http://www.ivoa.net/xml/UWS/v1.0"
+#uws_2_namespace = "http://www.ivoa.net/xml/UWS/v2.0"
+
+
+class UWS_1_Names(object): 
```

Comment:
According to PEP8, this should be `UWS1Names` (camel case). Maybe you can call this type of classes `Flavours` since I think `Names` can be misleading.